src/traces/sunburst/attributes.js

```diff
@@ -215,5 +215,19 @@ module.exports = {
     },
     sort: pieAttrs.sort,
 
+    root: {
```

Comment:
Thinking about a tree it might be better to use `trunk.color` that `root.color`.
@alexcjohnson what is your suggestion here?

Reply:
Also we have `leaf` attribute in `sunburst`. So let's place this attribute below that one please.

Reply:
I think `root` is OK - it's fairly common usage I think as the very first node in hierarchical structures, whereas `trunk` implies "the main branch" which may be a chain of multiple nodes.

Reply:
Alright. So let's keep `root` as is. Resolving this item...
@thierryVergult there are only a few unresolved comments below.